File: Latihan 2/Java/Product.java
/* Saya Silmi Aulia Rohmah mengerjakan Latihan 2 DPBO 2022 C1 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */

public class Product{

    //atribut private kelas product
    private int price = 0;
    private String idProduct = "";

    
    public Product(){ //konstruktor kosong

    }

    //metode getter dan setter
	public int getPrice() {
		return this.price;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public String getIdProduct() {
		return this.idProduct;
	}

	public void setIdProduct(String idProduct) {
		this.idProduct = idProduct;
	}

    //untuk print atribut kelas Product
    public void getInfoProduct(){

        System.out.println("ID Product    : " + getIdProduct());
        System.out.println("Price         : " + getPrice());
        
    } 

}

File: Latihan 2/C++/Product.cpp
/* Saya Silmi Aulia Rohmah mengerjakan Latihan 2 DPBO 2022 C1 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */


#include <iostream>
#include <string>

using namespace std;

class Product{ 

    private:
        // atribut class Product
        int price;
        string IdProduct;

    
    public:

        Product(){ //konstruktor kosong
        }

        // metode getter dan setter

        int getPrice(){
            return this->price;
        }

        void setPrice(int price){
            this->price = price;
        }

        string getIdProduct(){
            return this->IdProduct;
        }

        void setIdProduct(string IdProduct){
            this->IdProduct = IdProduct;
        }
        
        //untuk print atribut kelas Product
        void getInfoProduct(){
            cout << "ID Product    : " << getIdProduct() << endl;
            cout << "Price         : " << getPrice() << endl;
        }

        ~Product(){ //destruktor
        }
};

